# Follow trajectory behavior action definition

# Request
as2_msgs/TrajectorySetpoints trajectory         # trajectory to follow
---
# Result 
bool follow_path_success                        # false if failed to follow_path
---
# Feedback
geometry_msgs/TwistStamped actual_speed         # actual speed (m/s)
float32 actual_distance_to_next_waypoint        # distance to next waypoint (m)
uint16 remaining_waypoints                      # remaining waypoints